src/test/java/io/fabric8/maven/XMLFormatTest.should_ident_and_insert_line_break_on_major_sections.approved.txt

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+    <child />
```

Comment:
I think you should test that with multiple sections that have children in them.

Ideally, I would put the newline before and not after so that the first item has a new line before.

Reply:
This part is still broken, just wanted to share the API first :)